src/r_o_d_completion.lean

```diff
@@ -14,42 +14,42 @@ import for_mathlib.sheaves.stalk_of_rings -- for defining valuations on stalks
 
 variable {A : Huber_pair}
 
-open topological_space valuation Spv Spa
+open topological_space valuation Spv spa
 
```

Comment:
Does the actual *name* of this file need to be changed? I want to call it "A<T/s>" because that's what it's about.

Reply:
Let's do that in another PR